src/styles/calendar.scss

```
@@ -0,0 +1,127 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  max-width: ($box-side + 2) * $weekdays;
+
+  & :nth-child(-n+31) {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $box-side;
+    height: $box-side;
+    margin: 1px;
+    border: 1px solid black;
+
+    background-color: $background-color;
+  }
+
+  & :hover {
+    background-color: $background-color-hover;
+    transition-duration: 0.5s;
```

Це перенеси у звичайний стан блоку, щоб анімація була плавною в 2 сторони і краще вказувати тривалість для конкретних властивостей як мені сказали, а саме: 
```suggestion
    transition-duration: background-color 0.5s, transform 0.5s;
```